File: DriverTracker.Server/Models/LoginModel.cs
using System;
namespace DriverTracker.Server.Models
{
    public class LoginModel
    {
        public InputModel Input { get; set; }

        public LoginModel()
        {
        }

        public class InputModel
        {
            public String Email { get; set; }
            public String Password { get; set; }
            public bool RememberMe { get; set; }
        }
    }
}

File: DriverTracker.Client/drivertracker/src/app/login-model.ts
export class LoginModel {
    input: Input;

    constructor(email: string, password: string) {
        this.input.email = email;
        this.input.password = password;
        this.input.rememberMe = false;
    }
}

export class Input {
    email: string;
    password: string;
    rememberMe: boolean;
}